Mathlib/Algebra/Order/BigOperators/Ring/Finset.lean

```diff
@@ -125,22 +125,35 @@ section LinearOrderedCommSemiring
 variable [LinearOrderedCommSemiring R] [ExistsAddOfLE R]
 
 /-- **Cauchy-Schwarz inequality** for finsets. -/
```

Comment:
I think this could use an explanation in the docstring of the unusual statement.